README.md

```diff
@@ -156,7 +156,7 @@ we configured protractor to find them.
 
 Thereafter, run them with `npm run e2e`.
 
-That command first compiles, then simultaneously starts the Http-Server at `localhost:8080`
+That command first compiles, then simultaneously starts the Http-Server at `localhost:3000`
```

Comment:
it is not `http-server` anymore, even if lite-server is a http server, I guess we could remove the - in there.